Fix test failures caused by some build machines running more slowly than
expected, which manifest like this:

------
>       raise FailedHealthCheck(message)
E       hypothesis.errors.FailedHealthCheck: Data generation is extremely slow: Only produced 4 valid examples in 1.08 seconds (1 invalid ones and 0 exceeded maximum size). Try decreasing size of the data you're generating (with e.g.average_size or max_leaves parameters).
E       See http://hypothesis.readthedocs.org/en/latest/healthchecks.html for more information about this. If you want to disable just this health check, add HealthCheck.too_slowto the suppress_health_check settings for this test.

/gnu/store/b8d1r8bilvm3jkncgrpvmg3dni9cgcr1-python-hypothesis-3.1.0/lib/python3.5/site-packages/hypothesis/core.py:296: FailedHealthCheck
------

Patch copied from upstream source repository:

https://github.com/pimutils/vdirsyncer/commit/10490a12f13f03495e0945eb9d45d7aed9ab0a6c

From 10490a12f13f03495e0945eb9d45d7aed9ab0a6c Mon Sep 17 00:00:00 2001
From: Markus Unterwaditzer <markus@unterwaditzer.net>
Date: Sat, 18 Feb 2017 15:45:06 +0100
Subject: [PATCH] Unconditionally disable HealthCheck.too_slow

---
 tests/conftest.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/conftest.py b/tests/conftest.py
index e0a07d5..3afd7cd 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -26,10 +26,12 @@ def benchmark():
 else:
     del pytest_benchmark
 
+
+settings.suppress_health_check = [HealthCheck.too_slow]
+
 settings.register_profile("ci", settings(
     max_examples=1000,
     verbosity=Verbosity.verbose,
-    suppress_health_check=[HealthCheck.too_slow]
 ))
 settings.register_profile("deterministic", settings(
     derandomize=True,
